inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/UserServiceImpl.java

```diff
@@ -318,7 +318,8 @@ public Boolean delete(Integer userId, String currentUser) {
      */
     @Override
     public void login(UserLoginRequest req) {
-        String username = req.getUsername();
+        String username = req.getUsername().trim();
```

Comment:
Please delete this modification as verification logic has been added to the login page.